modules/Hoot/tools/clipDataset.js

```diff
@@ -193,7 +193,7 @@ export default class ClipDataset {
             let row         = d3.select( `#row-${ mapId }` ),
                 datasetName = row.select( '.datasetName' ),
                 outputName  = row.select( '.outputName' ),
-                folderId    = row.select( '.outputPath').property( '_value' );
+                folderId    = Hoot.folders.findBy( 'path', row.select( '.outputPath' ).property( 'value' ) ).id;
```

Comment:
The `_value` property should have the folder id.  Was this change correcting something else?

Reply:
Before this change, you couldn't change the destination path correctly. 

If the folder path provided when the modal became available would be the folder path the clipped dataset was sent to, regardless of what you had selected the path to be.

Reply:
Looking elsewhere in the code, I think it should be `.attr('_value')`
that might fix it.